tmt/steps/scripts/tmt-report-result

```diff
@@ -120,8 +120,16 @@ TESTNAME=${TESTNAME//\"/\\\"}
 TESTRESULT=$(echo "$2" | tr '[:upper:]' '[:lower:]')
 
 if [[ "$0" == *"rhts-report-result"* ]]; then
-    outputFile=$3
+    # The original outputFile ($3) is a `mktemp` file, but we can make the
+    # filename more predictable since `copy_outputfile_to_data_dir` puts it in
+    # separate folders
+    # TODO: This may be beakerlib specific, in which case maybe better to move
+    #   this to a specific beakerlib report script
```

Comment:
Another approach as hinted here is to move this logic into the beakerlib specific section and make another script wrapper for
https://github.com/teemtee/tmt/blob/40d62c52f6da7556f877374a8b56577e89bcd44b/tmt/frameworks/beakerlib.py#L20

Would prefer this option if there are no downsides to going with it, although handling the naming clash would still be a bit complicated.